backend/tests/test_b07_p7_ledger_cost_cache_audit.py

```
@@ -0,0 +1,685 @@
+            return "closed"
+        return "reserved"
+    if column == "status":
+        return "success"
+    if column == "breaker_state":
+        return "closed"
+    if column == "breaker_key":
+        return "llm-provider"
+    if column in {"reason", "routing_reason"}:
+        return "phase7_probe"
+    if column == "prompt_fingerprint":
+        return hashlib.sha256(f"probe:{marker}".encode("utf-8")).hexdigest()
```

Ruff (0.15.0)

[warning] Unnecessary UTF-8 `encoding` argument to `encode`

Remove unnecessary `encoding` argument

(UP012)